codalab/worker/dependency_manager.py

```diff
@@ -486,4 +486,7 @@ def interruptable_read(*args, **kwargs):
         else:
             with self._paths_lock:
                 self._paths.remove(dependency_state.path)
+            from codalab.lib import telemetry_util
```

Comment:
I think it should be OK to move this dependency out, to the top of the file. I don't think there's much overhead in importing it.

Reply:
@yiboliu can you take a look at this?